src/components/home/AllCollections.js

```diff
@@ -145,17 +73,7 @@ function AllCollections() {
   const [selectedLanguages, setSelectedLanguages] = useState([]);
 
   // state variable to control the side menu
-  const [sideBarStatus, setSideBarStatus] = useState(true);
-  const [gridParams, setGridParams] = useState({ sm: 12, md: 6, lg: 4, xl: 4 });
-
-  const closeSideBar = () => {
-    setSideBarStatus(false);
-    setGridParams(null);
-  };
-  const openSideBar = () => {
-    setSideBarStatus(true);
-    setGridParams({ sm: 12, md: 6, lg: 4, xl: 4 });
-  };
+  const [gridParams] = useState({ sm: 12, md: 6, lg: 4, xl: 4 });
```

Comment:
Shoud this be kept as state? It seems this can become a standalone static prop since `setGridParams` is not used anymore